dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java

```diff
@@ -78,9 +78,10 @@ private static void loadStarterSiteData() throws Exception{
 	private static void loadStarterSite() throws Exception{
 		
 	    loadStarterSiteData() ;
-	    
-		DbConnectionFactory.closeAndCommit();
-		
+
+        DbConnectionFactory.closeAndCommit();
+
+        new Task210304RemoveOldMetadataFiles().executeUpgrade();
```

Comment:
is this ok here or should be on the always run?

Reply:
why this here?